tools/c7n_index/c7n_index/metrics.py

```diff
@@ -47,14 +51,15 @@
 
 CONFIG_SCHEMA = {
     'type': 'object',
-    'additionalProperties': True,
-    'required': ['indexer', 'accounts'],
+    'additionalProperties': False,
+    'required': ['indexer', 'sqs'],
     'properties': {
         'indexer': {
             'oneOf': [
                 {
                     'type': 'object',
-                    'required': ['host', 'port', 'idx_name'],
```

Comment:
If `idx_name` is no longer a required property then I think the code here
https://github.com/salwandhruvdev/cloud-custodian/blob/e62d4f555f4234f596c8d92820c46b5cae9b45eb/tools/c7n_index/c7n_index/metrics.py#L180
has to change.

Reply:
Also validating this outside schema https://github.com/salwandhruvdev/cloud-custodian/blob/incoming-salwan-add-sqs-consumer/tools/c7n_index/c7n_index/metrics.py#L460-L464